### ISSUE TITLE
Should we remove dropout when training BYOL?

### ISSUE THREAD

**Issue by mathfinder** [lucidrains/vit-pytorch#23]
Should we remove dropout when training BYOL?

**Comment by lucidrains** [comment]
Wouldn't hurt!

**Comment by lucidrains** [comment]
@mathfinder Oops, I thought you meant add dropout. The truth is, the larger your attention network, the less dropout really matters, imo. If you are working with a rather small model, yeah, keep the dropout
